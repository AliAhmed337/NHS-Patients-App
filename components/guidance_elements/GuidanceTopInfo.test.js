import React from 'react';
import { shallow } from 'enzyme';
import { GuidanceTopInfo } from './index';

describe('Testing guidance info component', () => {

  it('renders correctly', () => {
    const wrapper = shallow(<GuidanceTopInfo />);
    expect(wrapper).toMatchSnapshot();
  });
});
