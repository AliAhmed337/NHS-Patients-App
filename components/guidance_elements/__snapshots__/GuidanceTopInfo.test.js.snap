// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing guidance info component renders correctly 1`] = `ShallowWrapper {}`;
